docs/content/components/container.mdx

```diff
@@ -2,30 +2,139 @@
 title: Container
 ---
 
+import { Box, Container, Text } from '@marigold/components';
+
 # Container
 
 ## Description
 
-Primitive layout component to embed other elements.
+Containers are used to constrain a content's width to the current breakpoint, while keeping it fluid. You can use `header`or `content`as content type and set the `maxWidth` to `small`, `medium` or `large`.
```

Comment:
We don't respect breakpoints here, I think 🤔

Maybe: width is based on used font and is optimized for readability